### PR TITLE
Bug 1209717 - Limit scopes used for try pushes

### PR DESCRIPTION
**Pull request by gregarndt**

### CODE REVIEW

**Comment by jonasfj** [comment]
r+

File: src/config/default.yml [hunk 1]
```diff
@@ -81,6 +81,19 @@ try:
...
+        - scheduler:*
+        - docker-worker:cache:tooltool-cache
+        - docker-worker:cache:tc-vcs
+        - docker-worker:cache:linux-cache
+        - docker-worker:capability:device:loopbackAudio
+        - docker-worker:capability:device:loopbackVideo
+        - docker-worker:image:taskcluster/*
+        - docker-worker:image:taskclusterprivate/phone-builder:*
+        - docker-worker:relengapi-proxy:tooltool.download.internal
+        - docker-worker:relengapi-proxy:tooltool.download.public
```
Comment (jonasfj) [inline src/config/default.yml]: I would consider: `docker-worker:cache:*`

**Comment by jonasfj** [comment]
still r+

**Comment by gregarndt** [comment]
@jonasfj do you see any issues with me just granting docker-worker:images:*?  I'm not sure if we really care about limiting these and also it causes an issue when people want to use their custom images on try to test things out (like their own private quay.io repo).

**Comment by selenamarie** [comment]
I heard from ahal that he couldn't use his quay.io images -- could we allow this? Any issues with quay.io/ahal/* or should we do specific images?

**Comment by gregarndt** [comment]
so I would like to refrain from adding user specific scopes into mozilla-taskcluster as it's already a mess.  What I'm thinking is we'll just allow any docker image I suppose, at least for now.  The two private images  (phone builder and upload_symbols) were already being allowed on try, so just allowing everything is no worse than before.  Any objections?  I can update this with the right scopes and deploy.

**Comment by gregarndt** [comment]
@jonasfj can I get a quick sanity check and then I'll merge and deploy.

**Comment by jonasfj** [comment]
lgtm

**Pull request merged**